source/_patterns/02-organisms/60-checkout/07-simple-donation-ui.mustache

```diff
@@ -0,0 +1,12 @@
+<div id="donation-edit" class="simple-donation-ui row">
+	<div id="donation-edit-label" for="donation-edit-amount" class="small-7 large-10 columns">
+		<span class="hide-for-large-up small-text">Optional Donation to<br class="lu"> Kiva's Operation Costs:</span>
```

Comment:
is the br necessary? and if so could it have a better class name? or maybe no class at all?

Reply:
that's a foundation utility class line breaks. got it out of the source html in https://styleguide.kiva.org/?p=atoms-typography